requirements.txt

```diff
@@ -1,6 +1,6 @@
 datasets==1.16.1
 en_core_web_sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.1/en_core_web_sm-2.3.1.tar.gz
-numpy==1.22.0
+numpy==1.19.5
```

Comment:
Reverted back because 3.9 works and M1 https://github.com/explosion/thinc/issues/474